### PR TITLE
Added endpoint to return nutritional value for selected recipes

### PR DESCRIPTION
**Pull request by mtdang**
When user selects meals, total nutritional values (calories, calories, fat, sugars, carbohydrates, cholesterol, protein) is returned.

### CODE REVIEW

File: src/controllers/recipes_controller.js [hunk 1]
```diff
@@ -71,9 +72,42 @@ module.exports.processRecipesJson = (req, res) => {
...
+                nutritions.sugars.unit = sugars.unit
+            }
+
+            nutritions.calories.amount += calories.amount
+            nutritions.fat.amount += fat.amount
+            nutritions.cholesterol.amount += cholesterol.amount
+            nutritions.carbohydrates.amount += carbohydrates.amount
+            nutritions.protein.amount += protein.amount
+            nutritions.sugars.amount += sugars.amount
+        });
```
Comment (silverAndroid) [inline src/controllers/recipes_controller.js]: Would there be any cases where the units would be different? (ie. grams and kilograms)

Comment (singhprincejeet) [reply src/controllers/recipes_controller.js]: Can you make an issue for that and link it here, so we dont forget :D

File: src/controllers/recipes_controller.js [hunk 2]
```diff
@@ -71,9 +72,42 @@ module.exports.processRecipesJson = (req, res) => {
 
     pythonProcess.stdout.on("end", () => {
         res.status(200).json(dataString.toString());
-    });
+    });
```
Comment (singhprincejeet) [inline src/controllers/recipes_controller.js]: remove this extra space please

File: src/controllers/recipes_controller.js [hunk 3]
```diff
@@ -71,9 +72,42 @@ module.exports.processRecipesJson = (req, res) => {
 
     pythonProcess.stdout.on("end", () => {
         res.status(200).json(dataString.toString());
-    });
+    });    
 };
 
+module.exports.planMeals = ({body: {recipeIds = null}}, res) => {
```
Comment (singhprincejeet) [inline src/controllers/recipes_controller.js]: Can you also add test cases for this?

File: src/controllers/recipes_controller.js [hunk 4]
```diff
@@ -74,6 +75,45 @@ module.exports.processRecipesJson = (req, res) => {
   });
 };
 
+module.exports.planMeals = ({ body: { recipeIds = null } }, res) => {
```
Comment (silverAndroid) [inline src/controllers/recipes_controller.js]: So, when a user adds a meal that they're planning to make (on the app), would I be passing all the recipes that have been planned or just the meals recently added?

**Pull request closed without merging**